store/pipelineRecord.go

```diff
@@ -35,16 +35,32 @@ func (d *DataStore) CreatePipelineRecord(pipelineRecord *api.PipelineRecord) (*a
 	return pipelineRecord, nil
 }
 
-// FindPipelineRecordsByPipelineID finds the pipeline record by pipelineID.
-func (d *DataStore) FindPipelineRecordsByPipelineID(pipelineID string) ([]api.PipelineRecord, error) {
+// FindPipelineRecordsByPipelineID finds the pipeline records by pipelineID.
+func (d *DataStore) FindPipelineRecordsByPipelineID(pipelineID string, queryParams api.QueryParams) ([]api.PipelineRecord, int, error) {
+	pipelineRecords := []api.PipelineRecord{}
 	query := bson.M{"pipelineID": pipelineID}
```

Comment:
`pipelineID` has been changed to `pipelineId` in PR #167. More test will be appreciated.

Reply:
fixed